support/map.h

```diff
@@ -2014,13 +2014,14 @@ Mono_Posix_ToUtimbuf (struct utimbuf *from, struct Mono_Posix_Utimbuf* to);
 
 
 struct Mono_Unix_UnixSignal_SignalInfo {
-	int   signum;
-	int   count;
+	int   signum;   // Read by signal handlers
+	int   count;    // Read and written by signal handlers
 	int   read_fd;
-	int   write_fd;
-	int   have_handler;
-	int   pipecnt;
-	void* handler;
+	int   write_fd; // Read by signal handlers
+	int   pipecnt;  // Read by signal handlers
+	int   pipelock; // Used by pipelock_ functions in signal.c. Read and written by signal handlers
+	int   have_restore_handler;
```

Comment:
restore_handler and have_restore_handler are simply renames. Let's remove this part.